CHANGES.md

```diff
@@ -1,3 +1,25 @@
+## edge-18.12.1
+
+* CLI
+  * **New** `linkerd routes` command displays per-route stats for any resource
```

Comment:
this item was part of `edge-18.11.3` ;)

Reply:
ooops :)

Reply:
I also added some emphasis on *any resource*